### ISSUE TITLE
Can't find GLeaflet.gwt.xml on classpath with version 1.2.5

### ISSUE THREAD

**Issue by yissachar** [johndevs/gradle-vaadin-plugin#477]
Currently on version 1.2.4 of the plugin. I have a dependency on v-leaflet:

```
compile "org.vaadin.addon:v-leaflet:0.5.7"
```

My configs are:

```
vaadin {
    version '7.7.7'
    useClassPathJar = false
    logToConsole = true
}

vaadinRun {
    server = 'jetty'
    serverPort = 7070
    openInBrowser = false
}
```

This all works fine on 1.2.4, but when I update to 1.2.5 compilation fails with the following error:

```
:compileJava
:vaadinUpdateWidgetset
:processResources UP-TO-DATE
:classes
:vaadinClassPathJar SKIPPED
Unable to find 'org/peimari/gleaflet/GLeaflet.gwt.xml' on your classpath; could be a typo, or maybe you forgot to include a classpath entry for source?
:vaadinCompile FAILED
```

**Comment by johndevs** [comment]
Yes, can confirm. Looks like a regression after c1fe9676d0d21f35b2c3ec51a0ab6130e190fb22.

**Comment by johndevs** [comment]
I see, `v-leaflet` has a dependency on `org.peimari:g-leaflet-draw`, `com.vividsolutions:jts` and `com.fasterxml.jackson.core:jackson-databind` which are not Vaadin addons and will now get filtered out. This hasn't been officially supported but has apparently worked due to the bug. This is related to #471 where we should start allowing passing non-vaadin addon dependencies to the client compiler classpath.

**Comment by johndevs** [comment]
This is no fixed in 1.2.6 so that you will need to add the ``v-leaflet`` dependency as a ``vaadinCompile`` dependency. For more information see https://github.com/johndevs/gradle-vaadin-plugin/wiki/Using-addons#addons-with-third-party-gwt-dependencies-since-126

**Comment by yissachar** [comment]
Excellent, works perfectly now, thanks!
